questions/33473-extreme-classic-13-coins-problem/template.ts

```diff
@@ -0,0 +1,37 @@
+type Weigh1 = {Left: 1 | 2 | 3 | 4 | 5 | 6, Right: 7 | 8 | 9 | 10 | 11 | 12}

```

Comment:
Is this the template or the solution? At first glance, I don't know where we should start.

Reply:
<img width="526" alt="image" src="https://github.com/type-challenges/type-challenges/assets/25894545/f75ed017-75e2-4751-805b-f74af3f530c2">
<img width="502" alt="image" src="https://github.com/type-challenges/type-challenges/assets/25894545/955385e3-7e2e-45ef-8d89-21313b49b518">
<img width="496" alt="image" src="https://github.com/type-challenges/type-challenges/assets/25894545/4d00a051-c3f4-41b2-9d14-fd0ff0e135e1">

Added 3 types aims to offer additional details about the situation.